source/money.d

```diff
@@ -315,6 +349,17 @@ struct currency(string currency_name, int dec_places = 4, roundingMode rmode = r
             static assert(0, "opCmp with such 'other' not implemented");
     }
 
+    int opCmp(T)(const T other) const if (isNumeric!T)
+    {
+        if (this.amount < other * __factor)
+            return -1;
+        else if (this.amount > other * __factor)
+            return 1;
+        else
+            return 0;
```

Comment:
The return 0 case is not covered by the unittest.

Reply:
done.